### DISCUSSION TITLE
Where is the source code of the DataTable with OverflowMenu storybook?

### DISCUSSION THREAD

**Discussion by nigellima** [carbon-design-system/carbon#9643] [Help] [answered]
Hello, everyone. 

I'm trying to find the source code of the storybook for this example here

https://react.carbondesignsystem.com/?path=/story/components-datatable--with-overflow-menu
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/14295280/132905431-104d872c-1719-4c60-8241-7ddb7f3eeb26.png">

In the story tab it is displaying the code for the DataTableSkeleton and I cant find it among the story files in the DataTable package source code. Can someone point me where this is coming from? Thanks

**Comment by tay1orjones** [comment, marked as answer]
Hey @nigellima, sorry for the confusion. I think it's this one?

https://github.com/carbon-design-system/carbon/blob/8ce0b4c0e5547837888b4050fa8630a7692df79b/packages/react/src/components/DataTable/DataTable-story.js#L160-L195

**Reply by nigellima** [reply]
That's the one. Thank you very much
